Applications/VpView/vpFileDataSource.cxx

```diff
@@ -128,6 +134,57 @@ void vpFileDataSource::setMonitoringEnabled(bool enable)
   }
 }
 
+bool add_mie4nitf_subdatasets (const QString &path, QStringList &list) {
+#ifdef VISGUI_USE_GDAL
+    GDALAllRegister();
+
+  GDALDataset *dataset = \
+        static_cast<GDALDataset *>(GDALOpen(path.toStdString().c_str(),
+              GA_ReadOnly));
+
+    if (!dataset)
+    {
+      qWarning().nospace()<< "GDAL could not load file." <<
```

Comment:
Why is `nospace()` needed here? Also, please put `<<` at the start of the next line, not the end of a line, and align with the prior `<<`.